scripts/backup

```diff
@@ -43,14 +44,13 @@ upstream_version=$(ynh_app_upstream_version)
 #=================================================
 ynh_script_progression --message="Backing up the main app directory..." --weight=1
 
-ynh_backup --src_path="$final_path"
-
 #=================================================
 # BACKUP THE NGINX CONFIGURATION
 #=================================================
 ynh_script_progression --message="Backing up nginx web server configuration..." --weight=1
 
 ynh_backup --src_path="/etc/nginx/conf.d/$domain.d/$app.conf"
+ynh_backup --src_path="/etc/nginx/conf.d/${server_name}.d/server_name.conf"
```

Comment:
You need to manage the case if this file don't exist. So just do a backup if this file exist.

Reply:
Is it what you meant?
If server_name, there is.well-know, you backup the file.
If no server_name, no backup.
If server_name but no .well-known, is not a situation that could exist if we merge the .well-known support at the same time that the server_name support.